lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -4695,7 +4695,7 @@ DefMacro('\framebox', '\@ifnextchar(\pic@framebox\@framebox', robust => 1);
 DefConstructor('\@framebox[Dimension][]{}',
   "?#mathframe(<ltx:XMArg enclose='box'>#inner</ltx:XMArg>)"
     . "(<ltx:text ?#width(width='#width') ?#align(align='#align')"
-    . " framed='rectangle' framecolor='#framecolor'"
+    . " framed='rectangle' framecolor='#framecolor' cssstyle='#cssstyle'"
```

Comment:
I am spiritually obliged to mention `css_style` looks better on every PR that uses the non-underscore variant :>

Reply:
`css_style`??? *so* boring...